README.rst

```diff
@@ -24,23 +24,31 @@ repository:
     **Deprecated**
     Ubuntu 18.04 supporting Python 2.7, 3.5, 3.6, 3.7.
 
-`readthedocs/build:6.0`
+`readthedocs/build:5.0`
     ``stable``
-    Ubuntu 18.04 supporting Python 2.7, 3.5, 3.6, 3.7, 3.8 and PyPy3.5-7.0.0.
+    Ubuntu 18.04 supporting Python 2.7, 3.6, 3.7 and pypy3.5-7.0.0.
     This is the **stable** image supported by Read the Docs.
 
-`readthedocs/build:7.0`
+`readthedocs/build:6.0`
     ``latest``
     Ubuntu 18.04 supporting Python 2.7, 3.5, 3.6, 3.7, 3.8 and PyPy3.5-7.0.0.
     This is the **latest** default image used for documentation builds and supported by Read the Docs.
 
-`readthedocs/build:8.0`
+`readthedocs/build:7.0`
     ``testing``
-    Ubuntu 20.04 supporting Python 2.7, 3.5, 3.6, 3.7, 3.8, 3.9.0rc1 and PyPy3.5-7.0.0.
+    Ubuntu 18.04 supporting Python 2.7, 3.5, 3.6, 3.7, 3.8, 3.9, and PyPy3.5-7.0.0.
     Available for public usage as **testing** image. You should expect some breaking changes here.
 
 .. _readthedocs/build: https://hub.docker.com/r/readthedocs/build/
 
+.. warning::
+
+   We are transitioning to a new way of building our Docker images, see
+   `the relevant design document <https://docs.readthedocs.io/en/stable/development/design/build-images.html>`_
+   for more information.
+   As a result, we will not update the current images
+   and will focus on implementing the new ones instead.
```

Comment:
Because this work isn't yet on our roadmap, I think we are still going to update the current images. Probably best to wait on this statement as well. In fact, I could see us doing a python 3.10 image, I don't think there should be anything blocking us from this.

Reply:
Thanks @agjohnson! I think it should be enough to remove this warning then? I'll revert this commit.

Reply:
Yeah, I suppose so, I'm not sure what else we'd mention here otherwise. Technically, nothing has changed yet. What was the underlying cause for updating this? Did we have a user question on state of the images?

Reply:
It leads tech-savvy users into thinking that the `testing` image ships Ubuntu 20.04, which is not the case at the moment.

Reply:
> In fact, I could see us doing a python 3.10 image, I don't think there should be anything blocking us from this.

Well... We never published the `8.0` because there is no clear path to upgrade images without being 100% sure that we are not breaking builds. Because of that, we started the discussion and created the design doc.

We can add py3.10 in our _current_ `testing` image as a workaround. However, that doesn't solve the root problem and it may cause others if we can't guarantee that all the versions are kept the same after image rebuild.

Also, we can't upgrade Rust to a newer version and be 100% sure that it won't break people's builds. Hopefully, not many users should be using `testing`, but... 🤷🏼

Reply:
I don't think there's _any_ way we can pull the rug from under the user's feet and not having any breakage.

An alternative would be: make users explicitly say which image version they want, with the number. Pros: we can guarantee that it doesn't break (and if it does, it's a bug, and not a feature), it works with the current scheme of things (we would just gradually phase out the names `stable`, `testing`, and such -- since these "moving tags" are the root of the problem), it gives us a way to move forward without fear of breaking stuff, and it's also forward compatible with any changes we do in how do we build such images. Cons: we might need to introduce some form of deprecation or support policy for people stuck in very old images.

How does that sound @humitos @agjohnson @ericholscher ?

Reply:
We've already had this conversation a million times and it's all trade offs. We should work on our design doc if we want to discuss it again, otherwise let's just update the README to reflect reality in this PR.

Reply:
Fair enough. This needs a ✅ review then :)